Resources/Private/BackendModule/src/Components/ConfigurationForm/LimitField.tsx

```diff
@@ -0,0 +1,33 @@
+import PureComponent from "../PureComponent";
+import PropTypes from "prop-types";
+import React from "react";
+
+export default class LimitField extends PureComponent {
+    static propTypes = {
+        configuration: PropTypes.object,
+        updateConfiguration: PropTypes.func
+    }
+
+    render() {
+        const {configuration, updateConfiguration} = this.props;
+        return (
+            <div className={'neos-control-group'}>
+                <label className={'neos-control-label'}>
+                    {this.translationService.translate('NEOSidekick.AiAssistant:AssetModule:configuration.limit.label', 'How many images per page?')}
```

Comment:
translationService fehlt in den PropTypes

Reply:
Ist inherited von unserem eigenen `PureComponent`, welches auf dem originalen `PureComponent` von React basiert und aber common code, wie zB. die Services, bereits mitbringt.

Reply:
Ja, hab ich erst später gesehen. War nicht ganz obvious, weil imports lies man normal ja nicht. Is okay.